src/renderer/component/publishForm/view.jsx

```diff
@@ -137,13 +139,14 @@ class PublishForm extends React.PureComponent<Props> {
   }
 
   handleChannelChange(channelName: string) {
-    const { name, updatePublishForm } = this.props;
+    const { name, updatePublishForm, myChannels } = this.props;
+    const form = { channel: channelName };
+    const namedChannelClaim = myChannels.filter(channel => channel.name === channelName)[0];
```

Comment:
use `Array.prototype.find()`? (instead of filter)

Reply:
Good call @skhameneh